OculusHeader hdr

uint32  device_id
uint16  device_type
uint16  part_number
uint32  status

OculusVersionInfo versin_info

uint32  ip_addr
uint32  ip_mask
uint32  connected_ip_addr

uint8   mac_addr0
uint8   mac_addr1
uint8   mac_addr2
uint8   mac_addr3
uint8   mac_addr4
uint8   mac_addr5

float64 temperature0
float64 temperature1
float64 temperature2
float64 temperature3
float64 temperature4
float64 temperature5
float64 temperature6
float64 temperature7
float64 pressure
